apps/client/src/pages/onboarding/onboarding-page.tsx

```diff
@@ -44,6 +42,46 @@ const stepSlugs = ['start', 'user', 'health', 'coverage', 'price', 'matching'];
 const completePath = routePath.REPORT;
 
 const OnboardingPage = () => {
+  const { Funnel, Step, go, currentStep, currentIndex } = useFunnel(
+    stepSlugs,
+    completePath,
+  );
+  const { openModal, closeModal } = useModal();
+
+  const navigate = useNavigate();
+  const handleGoHome = () => navigate(routePath.HOME);
+
+  const [basicInfoState, setBasicInfoState] =
+    useState<UserInfoStateProps>(initialState);
+  const [healthFirstSelected, setHealthFirstSelected] = useState<string[]>([]);
+  const [healthSecondSelected, setHealthSecondSelected] = useState<string[]>(
+    [],
+  );
+  const [coverageSelected, setCoverageSelected] = useState<number[]>([]);
+  const [priceRange, setPriceRange] = useState<[number, number]>([7, 15]);
+
+  const isUserValid = useUserInfoValid(basicInfoState);
+  const isHealthValid =
+    healthFirstSelected.length > 0 && healthSecondSelected.length > 0;
+
+  const handleCoverageSelectionChange = (selectedIndices: number[]) => {
+    setCoverageSelected(selectedIndices);
+  };
+
+  const stepValidationMap: Record<string, boolean> = {
+    start: true,
+    user: isUserValid,
+    health: isHealthValid,
+    coverage: coverageSelected.length > 0,
+  };
+
+  const isNextEnabled = stepValidationMap[currentStep] ?? true;
+
+  const progressIndex = Math.max(currentIndex - 1, 0);
+  const progressTotal = stepSlugs.filter(
+    (s) => s !== 'start' && s !== 'matching',
+  ).length;
```

Comment:
```suggestion
const excluded = ['start', 'matching'];
const progressTotal = stepSlugs.filter(s => !excluded.includes(s)).length;
```

이 코드로 변경 가능한가요? 고려되지 못한 이유는 뭐에요?

Reply:
```tsx
const stepSlugs = ['start', 'user', 'health', 'coverage', 'price', 'matching'];
```
처음에는 단순히 하드코딩을 없애야겠다는 생각만 해서 stepSlugs를 쓰는 데에 초점을 두고 작성했어요..
그런데 막상 코드를 수정하고 나서도 가독성이 여전히 떨어진다고 느껴서 찜찜했는데 
지욱님이 말하신 코드로 변경하는게 더 좋아보이네요... 이 코드로 변경 가능합니다

반영하겠습니다 !